livekit_internal.proto

```diff
@@ -107,6 +107,7 @@ message StartSession {
   bool recorder = 12;
   ClientInfo client = 11;
   string name = 13;
+  bool migrate = 14;
```

Comment:
is this needed? or could we determine from protocol_version + which node is receiving the message?

Reply:
removed